api/src/main/java/org/openmrs/module/fhir2/providers/util/FhirProviderUtils.java

```diff
@@ -33,6 +36,22 @@ public static MethodOutcome buildCreate(DomainResource resource) {
 		return buildWithResource(methodOutcome, resource);
 	}
 	
+	public static OperationOutcome buildDelete(OperationOutcome.IssueSeverity code, OperationOutcome.IssueType issueType,
+	        BaseServerResponseException exception) {
+		//created codeable concepts to be called by operationOutcome, codeableconcept only limited to two properties text and coding
+		CodeableConcept codeableConcept = new CodeableConcept().setText("MSG_DELETED").setText("informational")
+		        .setText("This resource has been deleted").setText("informational")
+		        .setText("http://terminology.hl7.org/CodeSystem/operation-outcome").setText("This resource has been deleted")
+		        .setText(exception.getMessage());
+		
+		OperationOutcome operationOutcome = new OperationOutcome();
+		operationOutcome.addIssue().setSeverity(OperationOutcome.IssueSeverity.ERROR).setCode(issueType)
```

Comment:
Depending on the server outcome, IssueSeverity can be either `ERROR, FATAL, WARNING `or `INFORMATION`. So this should be dynamic.
More information about OperationOutcome [here](https://www.hl7.org/fhir/operationoutcome.html)

Reply:
Thanks for clarifying about this, Anything to respond about my code above